index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alex's Portafolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <header class="top-menu">
+      <a class="toolbarlogo">Alejandro Puente</a>
+      <img
+        class="toolbar menu"
+        src="./icons/Enabled.png"
+        alt="collapsed menu icon"
+      />
+    </header>
+
+    <section>
+      <div class="welcomeinfo">
+        <h1 class="title">Hey There. I’m Alex Puente</h1>
+        <h3 class="subtitle">I am a Software Developer</h3>
+        <div class="paragraph">
+          I can help you build a product, feature or website. Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </div>
+      </div>
+      <ul class="contact-bar">
+        <li>
+          <a href="http://www.github.com">
+            <img
+              class="contactbar-img icon"
+              src="./icons/github.png"
+              alt="github logo"
+            />
+          </a>
+        </li>
+        <li>
+          <a href="http://www.linkedin.com">
+            <img
+              class="contactbar-img icon"
+              src="./icons/linkedin.png"
+              alt="linkedin logo"
+            />
+          </a>
+        </li>
```

Comment:
[REQUIRED] Please make sure to put list style here to none so it looks exactly like the Figma template
[REQUIRED] As the last reviewer mentioned fix the left margin beside your icons list
![image](https://user-images.githubusercontent.com/74744787/158874384-4ced519a-176f-4661-99be-41698af98108.png)